hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/terasort/PravegaSequenceOutputFormat.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.pravega.example.hadoop.terasort;
+
+import io.pravega.client.ClientFactory;
+import io.pravega.client.admin.StreamManager;
+import io.pravega.client.stream.EventStreamWriter;
+import io.pravega.client.stream.EventWriterConfig;
+import io.pravega.client.stream.ScalingPolicy;
+import io.pravega.client.stream.Serializer;
+import io.pravega.client.stream.StreamConfiguration;
+import io.pravega.example.hadoop.wordcount.PravegaOutputFormat;
+import io.pravega.example.hadoop.wordcount.PravegaOutputRecordWriter;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.mapreduce.RecordWriter;
+import org.apache.hadoop.mapreduce.TaskAttemptContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.Optional;
+
+/**
+ * A special PravegaOutputFormat to write events to Pravega stream in sequence:
```

Comment:
This needs to be explained. What is sequence referring to? Sequence of what? From the explanation below it sounds like this is referring to task id order.

Reply:
fixed, comments added as below:
 * The "sequence" here means:
 * All the keys in a segment are in order;
 * Segments are in order too by segment id - all the keys in segment with lower id are less than all the keys in segments with greater ids.